compiler/rustc_codegen_llvm/src/gotoc/mod.rs

```diff
@@ -97,13 +97,43 @@ impl<'tcx> GotocCtx<'tcx> {
         self.current_fn().mir().basic_blocks().indices().map(|bb| format!("{:?}", bb)).collect()
     }
 
+    fn should_skip_current_fn(&self) -> bool {
+        match self.current_fn().readable_name().as_str() {
+            // https://github.com/model-checking/rmc/issues/202
+            "fmt::ArgumentV1::<'a>::as_usize" => true,
+            // https://github.com/model-checking/rmc/issues/203
+            "<(dyn core::any::Any + core::marker::Send + 'static)>::downcast_ref" => true,
+            // https://github.com/model-checking/rmc/issues/204
+            "sys_common::thread_info::THREAD_INFO::__getit" => true,
+            // https://github.com/model-checking/rmc/issues/205
+            "panic::Location::<'a>::caller" => true,
+            // https://github.com/model-checking/rmc/issues/206
+            "core::sync::atomic::atomic_swap" => true,
+            // https://github.com/model-checking/rmc/issues/207
+            "core::slice::<impl [T]>::split_first" => true,
+            // https://github.com/model-checking/rmc/issues/208
+            "panicking::take_hook" => true,
+            // https://github.com/model-checking/rmc/issues/209
+            "panicking::r#try" => true,
+            _ => false,
+        }
+    }
+
     pub fn codegen_function(&mut self, instance: Instance<'tcx>) {
         self.set_current_fn(instance);
         let name = self.current_fn().name();
         let old_sym = self.symbol_table.lookup(&name).unwrap();
         assert!(old_sym.is_function());
         if old_sym.is_function_definition() {
             warn!("Double codegen of {:?}", old_sym);
+        } else if self.should_skip_current_fn() {
+            debug!("Skipping function {}", self.current_fn().readable_name());
+            let loc = self.codegen_span2(&self.current_fn().mir().span);
+            let body = Stmt::assert_false(
+                &format! {"The function {} is not curently supported by RMC", self.current_fn().readable_name()},
```

Comment:
Not sure if this is working as intended since you use brackets and not parentheses here:
```suggestion
                &format!("The function {} is not curently supported by RMC", self.current_fn().readable_name()),
```

Reply:
Fixed. Rust is fairly flexible with `{` and `(` in macros. But `(` does make more syntatic sense here